src/main/menu.ts

```diff
@@ -150,9 +150,12 @@ function getTasksMenu(): MenuItemConstructorOptions {
 function getShowMeMenuItem(
   key: string,
   item: string | Templates,
+  active_key: string | null,
 ): MenuItemConstructorOptions {
   if (typeof item === 'string') {
     return {
+      type: 'checkbox',
+      checked: key == active_key,
```

Comment:
Why we can't use `===` here?

```suggestion
      checked: key == activeKey,
```